rocAL_pybind/amd/rocal/decoders.py

```diff
@@ -419,7 +419,9 @@ def audio(*inputs, file_root='', file_list_path='', bytes_per_sample_hint=[0], s
         @param seed                     Random seed.
         @param stick_to_shard           The reader sticks to the data for it's corresponding shard when enabled
         @param shard_size               Number of files in an epoch
-        @return    Decoded audio.
+        @param last_batch_policy        Determines the handling of the last batch when the shard size is not divisible by the batch size. Check RocalLastBatchPolicy() enum for possible values.
```

Comment:
There's no RocalLastBatchPolicy enum in python side. Change it to Check types.py for possible values